test/src/e2e_vm_tests/test_programs/address_test/Forc.toml

```diff
@@ -1,9 +1,9 @@
 [project]
 author = "Fuel Labs <contact@fuel.sh>"
+entry = "main.sw"
 license = "Apache-2.0"
 name = "address_test"
-entry = "main.sw"
 
 [dependencies]
-std = { git = "http://github.com/FuelLabs/sway-lib-std", version = "v0.0.1" }
-core = { git = "http://github.com/FuelLabs/sway-lib-core", version = "v0.0.1" }
\ No newline at end of file
+core = { git = "http://github.com/FuelLabs/sway-lib-core" }
```

Comment:
This should not be merged without a pinned version. Can you tag a breaking release of `sway-lib-core` (i.e. `v0.1.0`), and use it here?

Reply:
done